### PR TITLE
fix(antd): disabled property of options of antd theme

### PR DESCRIPTION
**Pull request by KingMario**
### Reasons for making this change

The disabled properties of options of checkboxes and radio widgets are wrongly set.

This PR fixes the issues in the two widgets.

### Checklist

- [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://rjsf-team.github.io/react-jsonschema-form/docs/contributing) of the Markdown text I've added
- [x] **I'm adding or updating code**
  - [ ] I've added and/or updated tests. I've run `npm run test:update` to update snapshots, if needed.
  - [ ] I've updated [docs](https://rjsf-team.github.io/react-jsonschema-form/docs) if needed
  - [x] I've updated the [changelog](https://github.com/rjsf-team/react-jsonschema-form/blob/main/CHANGELOG.md) with a description of the PR
- [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,12 @@ should change the heading of the (upcoming) version to include a major version b
 
 -->
 
+# 5.18.5
+
+## @rjsf/antd
+
+- Fix disabled property of options in CheckboxesWidget and RadioWidgep ([#4216](https://github.com/rjsf-team/react-jsonschema-form/pull/4216))
+
```
Comment (heath-freenome) [inline CHANGELOG.md:24]: @KingMario if you resolve the conflict in this file then I will approve and merge

**Comment by heath-freenome** [comment]
@KingMario Still waiting for you to resolve the conflicts. Would love to get this into the next release that I hope to do in the next few days

File: CHANGELOG.md (hunk 1 above)
Comment (KingMario) [reply CHANGELOG.md:24]: Sorry for being late. The conflit is resolved.

**Comment by KingMario** [comment]
@heath-freenome sorry for being late. The conflict resolved.

**Comment by heath-freenome** [comment]
@KingMario I have no idea why your tests are now failing. Can you fix please?

**Comment by KingMario** [comment]
> @KingMario I have no idea why your tests are now failing. Can you fix please?

Let me see how to fix it.

**Comment by KingMario** [comment]
Hi @heath-freenome, I just updated the test snapshot according to the failed test. I hope this time the test passes.

BTW, when I ran `test:update`, the command failed. That's why I haven't checked `I've added and/or updated tests. I've run npm run test:update to update snapshots, if needed.` in this PR.

**Comment by KingMario** [comment]
@heath-freenome I see the test failed again.

I just uncommitted the test commit.

**Review by heath-freenome — APPROVED** [review]

**Pull request merged**